src/validate.js

```diff
@@ -169,12 +189,12 @@ var runnerSchema = Joi.object().keys({
     host: Joi.string().required(),
     gzip: Joi.boolean(),
     cache: Joi.boolean()
-  }),
+  }).when('sauce.launchSauceConnect', { is: true, then: Joi.forbidden().error(new Error('Can not use sauce connect tunnel with ngrok tunnel at the time')) }),
```

Comment:
Update error message to: `tunnel option cannot be set when launchSauceConnect flag is enabled`

Reply:
addressed